### DISCUSSION TITLE
Deploy on vercel

### DISCUSSION THREAD

**Discussion by Artem91S** [telegraf/telegraf#1918] [Q&A]
Dear All,
Please help with deploy my bot on vercel.
Below is my link to repo https://gitlab.com/ArtemS91/lesson-bot.git
Deploy is ok but the bot did not answer me.
Please help)

**Comment by MKRhere** [comment]
`bot.launch()` defaults to using long polling. You should probably use one of the [Webhook examples](https://github.com/feathers-studio/telegraf-docs/tree/master/examples/webhook).
